backend/mora/util.py

```diff
@@ -73,6 +73,8 @@
     2 * 60**2: DEFAULT_TIMEZONE,
 }
 
+logger = logging.getLogger(__name__)
```

Comment:
I'd just inline this one; you'll only be using it once, and ideally, we should use the Flask app logger instead. (Which might not work in this case…)

Reply:
We are not yet in a flask context, so flask is a no go.
Inline?
Do You mean logging.getLogger(__name__).warn(...)?